docs/static/schema.json

```diff
@@ -190,9 +195,14 @@
             "multiline"
           ]
         },
+        {
+          "required": [
+            "trimHeader"
+          ]
+        },
         {
           "skipPrerelease": [
-            "multiline"
+            "skipPrerelease"
```

Comment:
this change is not related to this PR goal but it looked like a "typo". please confirm this is the case.

Reply:
Great spot. This should be:

```json
{
  "required": [
     "skipPrerelease"
  ]
}
```

Reply:
I'll fix this shortly. The change looks great, and I don't want this to hold up merging this PR.